internal/core/window.rs

```diff
@@ -1035,7 +1035,7 @@ pub mod ffi {
         size: &IntSize,
     ) {
         let window_adapter = &*(handle as *const Rc<dyn WindowAdapter>);
-        window_adapter.set_size(crate::api::PhysicalSize::new(size.width, size.height).into());
+        window_adapter.window().set_size(crate::api::PhysicalSize::new(size.width, size.height));
```

Comment:
Has to change that because Window::set_size do resize the WindowItem in addition to to calling WindowAdapter::set_size

In fact, WindowAdapter::set_size shouldn't be called dirrectly. because then the two things gets out of sync.

Reply:
Ouch, good catch!